Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -3195,7 +3196,12 @@ void WebPageProxy::dispatchActivityStateChange()
 #if ENABLE(POINTER_LOCK)
     if (((changed & ActivityState::IsVisible) && !isViewVisible()) || ((changed & ActivityState::WindowIsActive) && !protectedPageClient()->isViewWindowActive())
         || ((changed & ActivityState::IsFocused) && !isViewFocused()))
-        requestPointerUnlock();
+        requestPointerUnlock([this, protectedThis = Ref { *this }](bool result) {
+            if (result) {
+                webContentPointerLockProcess()->send(Messages::WebPage::DidLosePointerLock(), webPageIDInProcess(*webContentPointerLockProcess().get()));
```

Comment:
Let's capture the result of webContentPointerLockProcess in a RefPtr and null check it.  No need to crash if state gets messed up.

Reply:
Added a null check